examples/twittermap/noah/src/main/scala/edu/uci/ics/cloudberry/noah/TwitterJSONTagToADM.scala

```diff
@@ -52,7 +54,6 @@ object TwitterJSONTagToADM {
       }
     }
   }
```

Comment:
Let's not make any unnecessary changes. Line 55 was removed. (doesn't have to be)